src/styles/main.scss

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
 }
+
+$block-width: 98px;
+$block-heigth: 98px;
+$entire-block: 100px;
+$gap: 1px;
+
+.block {
+  max-width: 726px;
+  min-width: 726px;
+  display: flex;
+  justify-content: left;
+  flex-wrap: wrap;
+  gap: $gap;
+  margin-top: 30px;
+
```

Comment:
You are missing a requirement
<img width="348" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/49764881/2f74c384-4fb8-42db-be2f-b143e4fc672f">